spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Cell.scala

```diff
@@ -36,6 +36,8 @@ abstract class Cell[T : ClassTag](val hiddensShape: Array[Int])
   (implicit ev: TensorNumeric[T])
   extends AbstractModule[Table, Table, T] {
 
+  var cell: AbstractModule[Activity, Activity, T]
```

Comment:
I added more scala docs for the `Cell` class and the `cell` variable to guide the user how to use it.